guides/common/modules/proc_upgrading-postgresql-on-the-external-database.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Upgrading_PostgreSQL_on_the_External_Database_{context}"]
+= Upgrading PostgreSQL on the external database
+
+If you use an external database, upgrade the external database and rectify the permissions.
```

Comment:
Can the "and rectify the permissions" part be hidden behind "ifndef::foreman-el" (or ifdef::katello_etc)? Otherwise the introduction says "do these two things" but then there is just one step to do.